cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -569,4 +579,25 @@ and then restart CDAP.
 #. Restart the CDAP processes::
 
      for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
+     
+If you are upgrading a secure Hadoop cluster, see the notes above about
+:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+
+.. _install-troubleshooting:
+
+Troubleshooting
+---------------------------------
+
+- If you are using a LinuxContainerExecutor, the ``cdap`` user needs to be present on all
+  Hadoop nodes.
+
+- If you are using a LinuxContainerExecutor, and the UID for the ``cdap`` user is less than
+  500, you will need to add the ``cdap`` user to the allowed users configuration for the
+  LinuxContainerExecutor in Yarn by editing the ``/etc/hadoop/conf/container-executor.cfg``
+  file::
+
+    sudo vi /etc/hadoop/conf/container-executor.cfg
```

Comment:
We can drop this line and assume they know how to edit the file.  Just leave the lines below.